standalone/inc/pup/PUPManager.cpp

```diff
@@ -70,7 +77,8 @@ void PUPManager::LoadConfig(const string& szRomName)
    for (auto& [key, pScreen] : m_screenMap) {
       PUPCustomPos* pCustomPos = pScreen->GetCustomPos();
       if (pCustomPos) {
-         PUPScreen* pParentScreen = GetScreen(pCustomPos->GetSourceScreen());
```

Comment:
Chicken and egg problem. GetScreen logs warnings if screens are referenced that don't exist (yet). However the code that populated the screens was already triggering these checks. So we directly access the m_screenMap instead.